app/helpers/participants_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ParticipantsHelper
   def hall_of_fame_title
-    params[:action] == "hall_of_fame" ? "Hall Of Fame" : "Organizer Of Fame"
+    params[:action] == "hall_of_fame" ? "HALL OF FAME_" : "ORGANIZER OF FAME_"
```

Comment:
Let's take the time to slowly refactor view text into the `pt.yml` translation file. What do you think?